disk-mapper/recoverproto/recover.proto

```diff
@@ -5,16 +5,16 @@ package recoverproto;
 option go_package = "github.com/edgelesssys/constellation/v2/disk-mapper/recoverproto";
 
 service API {
-    rpc Recover(stream RecoverMessage) returns (stream RecoverResponse) {}
+    rpc Recover(RecoverMessage) returns (RecoverResponse) {}
```

Comment:
Can you add some doc comments to this proto file?
You can follow [this example](https://github.com/edgelesssys/constellation/blob/83adbc621006671dc71ea60d3a58f19a4d642692/joinservice/joinproto/join.proto) from the dev-docs PR